### PR TITLE
feature/OB-17

### PR DESCRIPTION
**Pull request by csabbee**

### CODE REVIEW

File: app/actions/app-data.js [hunk 1]
```diff
@@ -29,6 +29,8 @@ export function fetchData(username) {
         dispatch(fetchingData());
 
         return getUserData(username)
-            .then(users => dispatch(receivedData(users)));
+            .then(users => {
+                dispatch(receivedData(users[0]));
```
Comment (ppseprus) [inline app/actions/app-data.js:33]: The BE should only return a single user anyway...

Comment (csabbee) [reply app/actions/app-data.js:33]: I know, that is a future fix

File: app/reducers/app-data.js [hunk 2]
```diff
@@ -13,8 +13,14 @@ export default function appData(state = defaultState, action) {
...
                 isFetching: false,
-                userData: action.data[0]
+                userData: action.data
             };
+        case 'ADD_ENTRY':
+            let userData = Object.assign({}, state.userData, { data: state.userData.data.concat(action.link) });
+            return {
+                isFetching: false,
+                userData
+            }
```
Comment (ppseprus) [inline app/reducers/app-data.js]: ;

**Review by ppseprus — APPROVED** [review]

File: app/actions/app-data.js (hunk 1 above)
Comment (ppseprus) [reply app/actions/app-data.js:33]: Just sayin'

**Pull request merged**